src/main/resources/feedbackQuestionTextSubmissionFormTemplate.html

```diff
@@ -1,7 +1,5 @@
 <div class="panel panel-default panel-body"
-    ${disabled} id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-    name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-    data-length-text-id="responseLength-${questionIndex}-${responseIndex}" data-recommended-text="${recommendedLength}">
+    data-is-session-closed="${isSessionClosed}" id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}" name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}" data-length-text-id="responseLength-${questionIndex}-${responseIndex}" data-recommended-text="${recommendedLength}">
```

Comment:
Note: also, don't change the order of the attributes while making the change here from what you've committed in https://github.com/TEAMMATES/teammates/pull/7811/commits/b73557e3e8bdf677ac0908cc001ff7601045b6c0 as otherwise you will have to run god mode again. 😅